android/brave_java_sources.gni

```diff
@@ -89,7 +89,7 @@ brave_java_sources = [
   "../../brave/android/java/org/chromium/chrome/browser/qrreader/BarcodeTrackerFactory.java",
   "../../brave/android/java/org/chromium/chrome/browser/qrreader/CameraSource.java",
   "../../brave/android/java/org/chromium/chrome/browser/qrreader/CameraSourcePreview.java",
-  "../../brave/android/java/org/chromium/chrome/browser/search_engines/settings/BraveSearchEngineAdapter.java",
+  "../../brave/android/java/org/chromium/chrome/browser/search_engines/settings/BraveBaseSearchEngineAdapter.java",
```

Comment:
Renamed this one to avoid ambiguity with existing class `brave/android/java/org/chromium/chrome/browser/settings/BraveSearchEngineAdapter.java`